andela/trime
# Show added project without refreshing

### What does this PR do?
Adds functionality to show the added project without refreshing when a user adds a project.

### Description of Task to be completed?
Add the added project to the list of projects when successfully added.

### How should this be manually tested?
Go to an org page, where there's a list of projects in the org. Use the add project button to add a project. The added project should show up in the list immediately it's added without refreshing.

### What are the relevant pivotal tracker stories?
[After adding a project, user should see the added project without refreshing](https://www.pivotaltracker.com/story/show/127625257)
